DirectEmployers/MyJobs
# [PD-3736] analytics search box not clear

## What's this PR do?
PR clears the searchbox text whenever you switch dimensions or drill down

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside any of the analytics applications

## How should this be manually tested?
Pull down repository. Go into any analytics application. Write some text in the search box and switch dimensions or drill down in the table and the text should clear from the search box. It will also clear after you search for something inside of the table.

## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
